draft-ietf-tls-exported-authenticator.md

```diff
@@ -270,6 +270,19 @@ A given authenticator can be validated by checking the validity of the
 CertificateVerify message given the authenticator request (if used) and recomputing the
 Finished message to see if it matches.
 
+# Empty Authenticator
+
+If, given an authenticator request, the endpoint does have an appropriate certificate
+or does not want to return one, it may construct an authenticated denial of existence
```

Comment:
s/may construct/constructs/

Reply:
"denial of existence" pairs poorly with "does not want to return one," since that implies that it's potentially lying.  This isn't DNSsec, where you actually are authenticating the denial of existence, but simply declining to answer the request and accepting the result of that.

Perhaps call this an "authenticated refusal" or a "denial of the request"?